.xo-config.json

```diff
@@ -0,0 +1,10 @@
+{
+	"overrides": [
+		{
+			"files": "source/{**,}.js",
+			"rules": {
+				"node/prefer-global/url": ["error", "never"]
```

Comment:
Use `// eslint-disable-next-line node/prefer-global`  instead.

Reply:
There will be 7 places with the comment and new files may not import URL properly. Are you totally sure? Maybe it's preferred to set `node/prefer-global/url` for the whole project (also tests)?

Reply:
> There will be 7 places with the comment

That's the point. So when it's fixed we don't forget to remove this workaround.

> Are you totally sure?

Yes.